projects/ion/src/lib/picker/date-picker/date-picker.component.scss

```
@@ -0,0 +1,11 @@
+@import '../../../styles/index.scss';
+
+.container-calendar {
+  max-width: 312px;
+  min-width: 312px;
+  margin: 8px 0px;
+  padding: 24px;
```

```suggestion
  padding: spacing(3);
```
Try to use `spacing` function for these spacing properties, like padding, margin, etc.